hotsos/core/config.py

```diff
@@ -112,6 +112,25 @@ def __init__(self):
                                         'log files, this is used to limit the '
                                         'logrotate history in days.'),
                            default_value=7))
+        self.add(ConfigOpt(name='allow_constraints_for_unverifiable_logs',
+                           description=('Search constraints use a binary '
+                                        'search that sometimes needs to '
+                                        'search backwards to find a last '
+                                        'known good line i.e. in log files '
+                                        "that contain lines that don't start "
```

Comment:
My nitpicking eyes were drawn to the "" here :-p

Reply:
thank brian, yeah that's cause of the ' in the string

Reply:
Oh yeah, didn't (or did not) even notice :)